snapcraft/project/__init__.py

```diff
@@ -15,3 +15,4 @@
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 from ._project import Project                          # noqa F401
+from ._project_info import ProjectInfo                 # noqa F401
```

Comment:
Eventually we want to get to the point where the Project can fill out the ProjectInfo based on its __init__, which means this won't need to be imported here. Would it make sense to just import the private module in the test instead of making it publicly available now?

Reply:
Fwiw `test_config` imports the private module by name today as "from snapcraft.project._project_info import ProjectInfo" for that reason.